packages/veritone-react-common/src/components/Parts/Buttons/PillButton/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { string, bool, func, objectOf, any } from 'prop-types';
+import classNames from 'classnames';
+import defaultStyles from './styles.scss';
+
+export default class PillButton extends Component {
+  static propTypes = {
+    label: string,
+    info: string,
+    style: objectOf(any),
```

Comment:
I don't have any strong reason why I have both style object & class name overwrite here. Just have extra time and want to provide both so that if someone wants to overwrite an existing style and doesn't want to use "!important"

Reply:
ok